SH4RE/SH4RE/Login/Tutorial.swift

```diff
@@ -0,0 +1,109 @@
+//
+//  Tutorial.swift
+//  SH4RE
+//
+//  Created by November on 2023-01-04.
+//
+
+import SwiftUI
+
+struct Tutorial: View {
```

Comment:
The backgorunds of these are supposed to be the "backgroundGrey". I know the images have a background to them, but you should be able to [change the image blend modes on the images](https://www.hackingwithswift.com/books/ios-swiftui/special-effects-in-swiftui-blurs-blending-and-more) to "Passthrough" which will make the white transparent.

Reply:
the backgrounds are the same colour, so i dont think this needs to be changed